src/controllers/restful/PaymentController.ts

```diff
@@ -410,25 +410,46 @@ export function getResponseErrorIfExists(
   if (faultBean === undefined) {
     return undefined;
   }
-  // Response is FAILED and additional information are provided by PagoPa
   return getErrorMessageCtrlFromPagoPaError(
+    faultBean.id,
     faultBean.faultCode,
-    faultBean.description
+    faultBean.description,
+    faultBean.originalFaultCode
   );
 }
 
 /**
  * Convert PagoPa message error (faultCode) to Controller message error (ErrorMessagesCtrlEnum) to send to BackendApp
  * A complete list of faultCode provided by PagoPa is available at
  * https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_nodo_2_1_0.pdf
+ * @param {string} faultId - Error Id provided by PagoPa to
  * @param {string} faultCode - Error code provided by PagoPa
  * @return {PaymentFaultEnum} Error code to send to BackendApp
  */
 export function getErrorMessageCtrlFromPagoPaError(
+  faultId: string,
   faultCode: string,
-  faultDescription: string | undefined
+  faultDescription: string | undefined,
+  originalFaultCode: string | undefined
 ): PaymentFaultEnum {
   switch (faultCode) {
+    case "PPT_EMESSO_DA_PAA":
```

Comment:
I checked the provided documentation at https://www.agid.gov.it/sites/default/files/repository_files/specifiche_attuative_nodo_2_1_0.pdf
and it reports `PPT_ERRORE_EMESSO_DA_PAA` (page 203) instead of `PPT_EMESSO_DA_PAA`
Could it be a mistyping?

Reply:
yes, mityping. correct value is PPT_ERRORE_DA_PAA.
updated documentation is https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/bozza/index.html